gulpfile.js

```diff
@@ -1,11 +1,19 @@
 'use strict';
 
+var path = require('path');
+
 const gulp = require('gulp');
 
 gulp.task('default', () => {
   return gulp.src([
-    'src/_scripts/**/*',
-    '!src/_scripts/{**/\_*,**/\_*/**}',
+    path.join('src','_scripts','/**/*'),
+    '!'+path.join('src','_scripts','/{**/\_*,**/\_*/**}'),
   ])
```

Comment:
you should be using `{cwd: src}` and not using `path.join()` on glob patterns. Glob patterns are regular expressions, not paths. fwiw, using the `path` module might lead to unexpected patterns that do not work. For example, on windows `path.join()` will use backslashes. In regular expressions, backslashes are not path separators, they are escape characters.